S/SCIP/build_tarballs.jl

```diff
@@ -7,16 +7,23 @@ version = v"0.1.2"
 
 # Collection of sources required to complete build
 sources = [
-    ArchiveSource("https://scip.zib.de/download/release/scipoptsuite-7.0.2.tgz", "f81b5a2c1c0eb949cf06bd50f42826e55284fa1269a6f28a92ac1a06d9c93a03")
+    ArchiveSource("https://scip.zib.de/download/release/scipoptsuite-7.0.2.tgz", "f81b5a2c1c0eb949cf06bd50f42826e55284fa1269a6f28a92ac1a06d9c93a03"),
+    GitSource("https://github.com/ds4dm/bliss.git", "debd9c4145747ffec38d020ed1e6a895e36f51f2"),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
 cd $WORKSPACE/srcdir
-cd scipoptsuite*
+cd bliss
+mkdir build
+cd build
+cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_BUILD_TYPE=Release ..
```

Comment:
You are not cross compiling bliss. At the very least you'll need something like `-DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN}` below. Perhaps take a look at the Bliss